frontend/packages/network-attachment-definition-plugin/src/selectors/index.ts

```diff
@@ -16,3 +16,9 @@ export const getConfigAsJSON = (
 export const getType = (config: NetworkAttachmentDefinitionConfig): string => {
   return _.get(config, 'type') === undefined ? null : config.type;
 };
+
+export const getDescription = (netAttachDef: NetworkAttachmentDefinitionKind): string => {
+  return _.get(netAttachDef, 'metadata.annotations.description') === undefined
```

Comment:
I had it the way it was because of Filip told me that it was the new way we are doing selectors to maintain type data. I went ahead and changed it. I can always change it back in a followup PR.